oscal-rest-service-app/src/main/java/com/easydynamics/oscalrestservice/api/SspController.java

```diff
@@ -23,13 +35,17 @@
 @RequestMapping(path = "/oscal/v1")
 @RestController
 public class SspController extends BaseOscalController<SystemSecurityPlan> {
+  private final Logger logger = LoggerFactory.getLogger(this.getClass());
+  private final OscalObjectMarshaller<ImplementedRequirement> oscalSspImplReqtMarshaller;
```

Comment:
```suggestion
  private final OscalObjectMarshaller<ImplementedRequirement> oscalSspImplReqMarshaller;
```

Maybe change this to match the naming convention of "Impl Req" used elsewhere

Reply:
Addressed